Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM qazbnm456/fbctf_base:latest
+MAINTAINER Lobsiinvok lobsiinvok@tdohacker.org
+
+ENV HOME /root
+ARG DOMAIN
+ARG MODE=dev
+CMD ["/sbin/my_init"]
+
+WORKDIR $HOME
+RUN apt-get update
+ADD . $HOME
+RUN chown root:root $HOME
+
+RUN ./extra/provision.sh $MODE `pwd`
+RUN [ "$MODE" = "prod" ] && rm /etc/nginx/sites-enabled/fbctf.conf || true
+
+COPY extra/http_fbctf.conf /etc/nginx/sites-available/http_fbctf.conf
+COPY extra/https_fbctf.conf /etc/nginx/sites-available/https_fbctf.conf
+RUN sed -i 's/domain.com/'$DOMAIN'/g' /etc/nginx/sites-available/http_fbctf.conf \
+  && sed -i 's/domain.com/'$DOMAIN'/g' /etc/nginx/sites-available/https_fbctf.conf
+
+RUN mkdir -p /etc/my_init.d
+COPY extra/setup_lets_encrypt.sh /etc/my_init.d/01_setup_lets_encrypt.sh
+COPY extra/service_startup.sh /etc/my_init.d/02_service_startup.sh
+COPY extra/import_empty_db.sh /etc/my_init.d/03_import_empty_db.sh
+RUN [ "$MODE" = "dev" ] && rm /etc/my_init.d/01_setup_lets_encrypt.sh || true
+RUN [ "$MODE" = "prod" ] && sed -i 's/domain.com/'$DOMAIN'/g' /etc/my_init.d/01_setup_lets_encrypt.sh || true
```

Comment:
I think many people will be terminating TLS at their reverse proxy. Is it possible to skip all of the TLS stuff inside of the container?